Fastfile.private

```diff
@@ -239,10 +239,10 @@ reflected in `Info.plist` during building."
       # http://krausefx.com/blog/download-dsym-symbolication-files-from-itunes-connect-for-bitcode-ios-apps
       include_bitcode: false
     }
-    if project_file.include? ".xcodeproj" do
-      params[:project] = project_file
+    if project_file.include? ".xcworkspace" do
+      params[:workspace] = project_file
     else
-      params[:scheme] = project_file
+      params[:project] = project_file
     end
```

Comment:
I'd add a new `else` to handle if there's no `. xcworkspace` and no `. xcodeproj`, which means the project is misconfigured. In this case you can do `UI.error('No Xcode project file found')` or something like this.